docs/DeveloperGuide.md

```diff
@@ -479,13 +491,14 @@ The `EditCommand` class is responsible for editing current patient's information
 #### Specifications
 
 * EditCommand, as defined by the `EditCommand` class, contain parameters which consists of:
-    *  `INDEX` integer,
-    *  `patientHospitalId` integer,
-    *  `name`, `preferredName` String with only alphabets character,
+    *  `INDEX` positive integer from 1 onwards, indicating the index of patient in the PatientSync list,
+    *  `patientHospitalId` non-negative numeric characters which uniquely identifies the patient,
```

Comment:
maybe it might be better to say  non-negative numeric string instead as people might be wondering why not use non-negative integer?

Reply:
Agree!